tests/framework/web/stubs/ImageModel.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace yiiunit\framework\web\stubs;
+
+use yii\base\Model;
+
+class ImageModel extends Model
```

Comment:
There is `yiiunit\framework\web\stubs\ModelStub` already.
There is no reason to add extra class here.